src/plugins/pol/CMakeLists.txt

```diff
@@ -3,22 +3,21 @@ include_directories(${GPUI_INCLUDE_DIRS})
 
 find_package(Qt5 COMPONENTS Core REQUIRED)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Why set standard here explicitly?

Reply:
I explicitly specified CXX 17 because it was not selected, although it is available elsewhere. So I had to explicitly specify